src/utils.ts

```diff
@@ -184,6 +184,52 @@ function getGitHubToken(): string {
   return core.getInput(c.INPUT_GITHUB_TOKEN)
 }
 
+export async function findExistingPrCommentId(
```

Comment:
Minor nit, for consistency with other PR-related utils:
```suggestion
export async function findExistingPRCommentId(
```